spyder/plugins/editor/widgets/editor.py

```diff
@@ -622,7 +622,17 @@ def show_in_external_file_explorer(self, fnames=None):
         """Show file in external file explorer"""
         if fnames is None:
             fnames = self.get_current_filename()
-        show_in_external_file_explorer(fnames)
+        try:
+            show_in_external_file_explorer(fnames)
+        except FileNotFoundError as error:
+            file = str(error).split("'")[1]
+            if "xdg-open" in file:
+                msg_title = _("Warning")
+                msg = _("Spyder can't show the file in the External file "
+                        "explorer because xdg-open is only available in "
+                        "Ubuntu and Debian.")

```

Comment:
```suggestion
                msg = _("Spyder can't show this file in the external file "
                        "explorer because the <tt>xdg-utills</tt> package is "
                        "not only available in your system.")
```

What about if we change the message like this?

Reply:
The thing is probably few people know about the existence of `xdg-open` and how it works. But this way we can let them know what they need to do to fix the problem (i.e. go and install `xdg-utils`) even if they don't understand the cause of it.